iotdb-cli/src/main/java/org/apache/iotdb/cli/tool/ImportCsv.java

```diff
@@ -423,6 +409,31 @@ public static void importCsvFromFile(String ip, String port, String username,
         connection.close();
       }
     }
+  }
 
+  private static void importFromSingleFile(File file) {
+    if (file.getName().endsWith(FILE_SUFFIX)) {
+      loadDataFromCSV(file, 1);
+    } else {
+      System.out.println(
+              "[WARN] File " + file.getName() + " should ends with '.csv' "
+                      + "if you want to import");
```

Comment:
It's better to convert all Syso to LOGGER. In this case, it's better to use LOGGER.warn()